Stellar-ledger.x

```diff
@@ -488,13 +488,27 @@ case 3:
 // This struct groups together changes on a per transaction basis
 // note however that fees and transaction application are done in separate
 // phases
-struct TransactionResultMeta
+struct TransactionResultMetaV0
```

Comment:
Is this change actually necessary? I think leaving this as is would decrease the amount of the necessary downstream changes.

Reply:
Removed